Visualizations-TablePanel: added YouTube link to Table Panel (#84533)
grafana/grafana: docs/sources/panels-visualizations/visualizations/table/index.md

# Table

{{< youtube id="PCY7O8EJeJY" >}}

> 

Author: I placed it like the others, but I agree. Let's put it after 😄

> 

Reviewer: I should probably go back and look at the other ones that have been added again. I know one got merged before I had a chance to review it.

> 

Author: @imatwawana 
Should we merge this before I add the descriptions mentioned before the video?

> 

Reviewer: It doesn't look like you've made the original change that I asked for to move the video to what is currently line 33/36. Do you want me to do that for you and then you can add the lead-in sentence?

> 

Author: Oh! Yes please.
So sorry, I understood you'd do the change, I apologize for the confusion.
Please do and I will send the update after that :)

{{< figure src="/static/img/docs/tables/table_visualization.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}

> The following video provides a visual walkthrough of the options you can set in a table visualization. If you want to see a configuration in action, check it out in the video:

Reviewer: ```suggestion
The following video provides a visual walkthrough of the options you can set in a table visualization. If you want to see a configuration in action, check out the video:
```

> The following video will give you a visual walkthrough over the configurations and settings you can set in a table panel. If you get stuck not knowing where is a setting in the documentation or want to see a configuration in action, check it out in the video below.

Reviewer: I took out the "if you get stuck in the documentation" part because I don't think we want to suggest that people are going to get stuck in the docs. Otherwise, I've just made some style fixes.

```suggestion
The following video provides a visual walkthrough of the configurations and settings you can set in a table visualization. If you want to see a configuration in action, check it out in the video:
```
